1.3/Dockerfile

```diff
@@ -1,13 +1,13 @@
 
-FROM erlang:19.0-rc2
+FROM erlang:18.3.3
```

Comment:
Good point, I have updated the Dockerfile. and this way once 18.3.4 is available on the official registry, Elixir image should pick up automagically.